apps/src/javalab/JavalabEditorTabMenu.jsx

```diff
@@ -2,12 +2,13 @@ import React, {Component} from 'react';
 import PropTypes from 'prop-types';
 import JavalabDropdown from './components/JavalabDropdown';
 import javalabMsg from '@cdo/javalab/locale';
-
+import onClickOutside from 'react-onclickoutside';
+import Radium from 'radium';
```

Comment:
We are trying to move away from Radium, and I'm not sure you need it here. (See https://docs.google.com/document/d/1Y3uK_iYMhTUaCI6yIDwOAMprIJCuzXSs2fECQggwp60/edit#heading=h.qxsunektx9id for the proposal about Radium––this proposal was passed, but it's not clear yet exactly how this work is going to get done.)

It looks like we do use Radium with onClickOutside in most places in our code, but I see https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/watchers/AutocompleteSelector.jsx#L5, which makes me think you might be able to get away with simply export default onClickOutside(JavalabEditorTabMenu); at the bottom ... does that work?

Reply:
Thanks so much for the feedback and background/context doc, Meg! I removed Radium and click outside behavior still works fine.

Reply:
Excellent! Glad it worked 👍